voidcrew/mapping/docking_port/_docking.port.dm

```diff
@@ -1,13 +1,68 @@
+/**
+ * The main docking port that all voidcrew ships should be using.
+ */
 /obj/docking_port/mobile/voidcrew
-	width = 12
-	dwidth = 5
-	height = 7
+	launch_status = UNLAUNCHED
+
+	/// Makes sure we dont run linking logic more than once
+	VAR_PRIVATE/cached_z_level
+	var/z_levels_above = 0
+	var/z_levels_below = 0
```

Comment:
Are these ever set to be anything other than 0 anywhere? Ctrl+F doesn't show anything

Reply:
for use later, will be needed for ships with multi-zs